src/Operators/coriolis.jl

```diff
@@ -0,0 +1,10 @@
+####
+#### Coriolis terms following Marshall et al. (1997) equation (25).
+####
+
+@inline fv̅ʸ(i, j, k, grid, v, f) = f * ℑy_aca(i, j, k, grid, v)
+@inline fu̅ˣ(i, j, k, grid, u, f) = f * ℑx_caa(i, j, k, grid, u)
+
+@inline fv(i, j, k, grid, v, f) = ℑx_faa(i, j, k, grid, fv̅ʸ, v, f)
+@inline fu(i, j, k, grid, u, f) = ℑy_afa(i, j, k, grid, fu̅ˣ, u, f)
```

Comment:
Why do we use the intermediate functions? This code was not very clear to me. Since we have double interpolation, can't we just use those operators directly in `coriolis.jl`?

I wonder if its dangerous to have files with the same name in different submodules. It is nice and clean to use `Operators/coriolis.jl`, but if this file is accidentally moved one level up, it will overwrite `src/coriolis.jl`. What do you think about that?

Reply:
Ah, I had volume factors in there that Jean-Michel pointed out weren't necessary for vertically stretched grid. I've switched to using the double interpolation operator.

I'm on board with unique file names. Then that file could become `coriolis_operators.jl` and we'll also have `difference_operators.jl`, `grid_utils.jl`, etc.

Reply:
I still don't think we should mix physics with low-level finite volume operators (eg, the current logic of `TurbulenceClosures` and `buoyancy` should probably apply to `coriolis` as well); but we can discuss how that works in the future.

Reply:
Hmmm, then maybe we can get rid of the `coriolis_operators.jl` if it makes sense to move them elsewhere, although there are multiple ways to discretize the Coriolis operator at the low-level so conserve energy or other properties (similarly with the other operators) so it might be nice to keep them in the `Operators` submodule.